eZ/Publish/Core/MVC/Symfony/Cache/Http/SignalSlot/TrashSlot.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot;
+
+use eZ\Publish\Core\SignalSlot\Signal;
+
+/**
+ * A slot handling TrashSignal.
+ */
+class TrashSlot extends PurgeAllHttpCacheSlot
```

Comment:
Is the naming off or does it actually purge all http cache and basicly DDOS the site? :)